site/gatsby-site/src/components/discover/parseURL.js

```diff
@@ -0,0 +1,74 @@
+import { parse } from 'query-string';
+import { decodeQueryParams } from 'use-query-params';
+
+const convertStringToRefinement = (obj) => {
+  const stringKeys = [
+    'source_domain',
+    'authors',
+    'submitters',
+    'incident_id',
+    'flag',
+    'classifications',
+    'is_incident_report',
+    'tags',
+    'language',
+  ];
+
+  let newObj = {};
+
+  for (const attr in obj) {
+    if (stringKeys.includes(attr) && obj[attr] !== undefined) {
+      // TODO: we have to make the other namespaces available here too
```

Comment:
We should add an issue to don't forget this TODO

Reply:
Opened https://github.com/responsible-ai-collaborative/aiid/issues/2381 to track this.